Makefile

```diff
@@ -346,7 +346,10 @@ TEST_PACKAGES_LINUX := \
 	testing/fstest \
 	text/template/parse
 
-TEST_PACKAGES_DARWIN := $(TEST_PACKAGES_LINUX)
+TEST_PACKAGES_LINUX := \
+	$(TEST_PACKAGES_LINUX) \
```

Comment:
Should this be `TEST_PACKAGES_DARWIN` ?

Reply:
You are correct, thanks for spotting this! I've pushed a fix.